docs/usage/PRODUCT_SYNC.md

```diff
@@ -59,14 +59,12 @@ order for the sync to resolve the actual ids of those references, those `key`s h
 final ProductSyncOptions productSyncOptions = ProductSyncOptionsBuilder.of(sphereClient).build();
 ````
 
-The options can be used to provide additional optional configuration for the sync as well:
+Additional optional configuration for the sync can be configured on the `ProductSyncOptionsBuilder` instance, according to your need:
 - `errorCallBack`
-a callback that is called whenever an event occurs during the sync process that represents an error. Currently, these 
-events.
+a callback that is called whenever an event occurs during the sync process that represents an error.
 
```

Comment:
a callback that is called whenever an error event occurs during the sync process.